integration_tests/build_test_containers.sh

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env bash
 
-if [[ "$(docker images -q d3a-tests 2> /dev/null)" == "" ]]; then
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
```

Comment:
Hmm, I do not necessarily disagree here, but wouldn't this cause problems if we decide to run this on Travis ? (because it is a Linux OS but does not use sudo - it is usually disabled in our jobs)

Reply:
hach, yes, you are right. Will revert it then. Is a remnant of me trying to make it run on my linux machine.